tests/test_dataset.py

```diff
@@ -495,14 +495,14 @@ def test_apply_symop_roundtrip(mtz_by_spacegroup):
     dataset = rs.read_mtz(mtz_by_spacegroup)
     for op in dataset.spacegroup.operations():
         applied = dataset.apply_symop(op)
-        back = applied.hkl_to_asu()
```

Comment:
We should try to resurrect this test when we do the refactor, but this is okay with me for now.